stack-9.0.1.yaml

```
@@ -36,12 +36,8 @@ extra-deps:
 - bytestring-encoding-0.1.0.0@sha256:460b49779fbf0112e8e2f1753c1ed9131eb18827600c298f4d6bb51c4e8c1c0d,1727
 - dependent-map-0.4.0.0@sha256:ca2b131046f4340a1c35d138c5a003fe4a5be96b14efc26291ed35fd08c62221,1657
 - dependent-sum-0.7.1.0@sha256:0e419237f5b86da3659772afff9cab355c0f8d5b3fdb15a5b30e673d8dc83941,2147
-- extra-1.7.9
 - floskell-0.10.5
```

extra-1.7.9 has a breaking change so no way
```suggestion
- extra-1.7.9
- floskell-0.10.5
```